airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -983,9 +984,11 @@ public AutoCloseableIterator<AirbyteMessage> augmentWithStreamStatus(final Confi
         new io.airbyte.protocol.models.AirbyteStreamNameNamespacePair(airbyteStream.getStream().getName(), airbyteStream.getStream().getNamespace());
     final var starterStatus =
         new StreamStatusTraceEmitterIterator(new AirbyteStreamStatusHolder(pair, AirbyteStreamStatusTraceMessage.AirbyteStreamStatus.STARTED));
+    final var runningStatus =
+        new StreamStatusTraceEmitterIterator(new AirbyteStreamStatusHolder(pair, AirbyteStreamStatus.RUNNING));
```

Comment:
I guess we've decided to add a 'running' state :-)  Maybe we shall file another ticket to add this state to other certified db sources.

Reply:
This is just to avoid error. I'll revert this and update test instead.